CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ All notable changes to this project will be documented in this file based on the
 
 ### Bugfixes
 - Function score query: corrected the `score_method` `average` to `avg` #975 
+- Set `json_decode()` assoc parameter to true in `Elastica\Response` #1005
```

Comment:
BIG_INT_* setting converts integers bigger than PHP_INT_MAX to a string, so yes, the response will be different, but it is the only way to deal with this without overflowing the integer.